Content.Server/StationEvents/MeteorSchedulerSystem.cs

```diff
@@ -13,12 +15,19 @@ namespace Content.Server.StationEvents;
 public sealed class MeteorSchedulerSystem : GameRuleSystem<MeteorSchedulerComponent>
 {
     [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+    [Dependency] private readonly IConfigurationManager _cfg = default!;
+
+    private TimeSpan _meteorMinDelay;
+    private TimeSpan _meteorMaxDelay;
 
     protected override void Started(EntityUid uid, MeteorSchedulerComponent component, GameRuleComponent gameRule, GameRuleStartedEvent args)
     {
         base.Started(uid, component, gameRule, args);
 
-        component.NextSwarmTime = Timing.CurTime + RobustRandom.Next(component.MinSwarmDelay, component.MaxSwarmDelay);
+        _cfg.OnValueChanged(CCVars.MeteorSwarmMinTime, f => { _meteorMinDelay = TimeSpan.FromMinutes(f); }, true);
+        _cfg.OnValueChanged(CCVars.MeteorSwarmMaxTime, f => { _meteorMaxDelay = TimeSpan.FromMinutes(f); }, true);
+
```

Comment:
this needs to be in the Initialize declaration or else you're going to create duplicated subscriptions.